change/@office-iss-react-native-win32-22bcae82-8c54-41e1-88f1-6aeb9deafbe3.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

Comment:
I believe the type needs to change here?

Reply:
Yup, thanks!